playbooks/roles/filebeat/tasks/main.yml

```diff
@@ -0,0 +1,53 @@
+---
+
+- name: Install apt package dependencies.
+  apt:
+    name: apt-transport-https
+    state: present
+
+- name: Add Elastic's apt key
+  apt_key:
+    url: https://artifacts.elastic.co/GPG-KEY-elasticsearch
+    state: present
+
+- name: Add Elastic's apt repository
+  apt_repository:
+    repo: 'deb https://artifacts.elastic.co/packages/6.x/apt stable main'
+    state: present
+    update_cache: yes
+
+- name: Install Filebeat
+  apt:
+    name: filebeat
+    state: present
+
+- name: Install Filebeat configuration
+  template:
+    src: filebeat.yml.j2
+    dest: "{{ filebeat_config_dir }}/filebeat.yml"
```

Comment:
Note: we don't really need this var because it'll be `/etc/filebeat` by default through the apt package install, but since I already added it I can just leave it.

Reply:
Yeah, I didn't mean this as a strict policy.  It's more a habit I got into – making everything configurable just doesn't add much value, and in some case hurts readability, and is more work to set up, so I just only do it if I have to.

Reply:
:+1: